src/subscription.ts

```diff
@@ -1039,6 +1050,14 @@ export class Subscription extends EventEmitter {
       delete formatted.pushEndpoint;
     }
 
+    if (metadata.oidcToken) {
+      formatted.pushConfig = {
+        ...formatted.pushConfig,
+        oidcToken: metadata.oidcToken,
+      };
+      delete formatted.oidcToken;
```

Comment:
I'm relatively green in this codebase, but mind clarifying why we need to delete the key off the `formatted` object?

Reply:
It does seem to already be happening above it for `formatted.pushEndpoint` and as well...

Reply:
Yeah, that was my take as well, oidcToken is just being treated like what's above. My main question is on the sort of systemic goodness of the patch - I don't understand the gestalt of the library enough to say yay or nay yet.

Reply:
I believe the changes here just expose access to the underlying API call, and is already exposed in other client libraries. With the exception of deleting the key (is this for memory management?) this is LGTM.